opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp

```diff
@@ -558,6 +566,14 @@ class WetGasPvt
             const Evaluation& fPrime = ((saturatedOilVaporizationFactor(regionIdx, temperature, pSat + eps) - Rv) - f)/eps;
 
             const Evaluation& delta = f/fPrime;
+
+            if ( Opm::MathToolbox<Evaluation>::isnan(delta) ) {
```

Comment:
You can use `Opm::isnan(delta)` or better `!Opm::isfinte(delta)` instead of this. the first just an alias, the second variant also catches infinite values.